rubix-core/src/main/java/com/qubole/rubix/core/CachingInputStream.java

```diff
@@ -243,11 +249,11 @@ public void run()
         RemoteReadRequestChain remoteReadRequestChain = null;
         CachedReadRequestChain cachedReadRequestChain = null;
         NonLocalReadRequestChain nonLocalReadRequestChain = null;
```

Comment:
Not needed to declare ` NonLocalReadRequestChain nonLocalReadRequestChain` here, can remove it now